modules/upgrade-cluster-version-parameters.adoc

```diff
@@ -0,0 +1,179 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrading-aws.adoc
+
+[id='upgrade-cluster-version-parameters-{context}']
+= ClusterVersion paramaters
+
+The Cluster Version Operator (CVO) uses the following parameters to manage
+cluster versions and upgrades.
+
+.ClusterVersion parameters
+
+[cols="2,8a",options="header"]
+|===
+|Parameter |Description
+
+|`ClusterVersionSpec`
+|The specification that describes the cluster version state that you want. The
+CVO updates your cluster components to the specified versions.
+
+|`ClusterVersionSpec.clusterID`
+|Required. The unique RFC4122 UUID value that identifies your cluster.
+`xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` hexadecimal value
+
+|`ClusterVersionSpec.desiredUpdate`
+|Optional. The cluster version that you want. If the current and intended
+version do not match, an upgrade starts. Set only a version from the list of
+recommended updates or your upgrade might fail.
+If an update exists with a version in the `availableUpdates` or history,
+you can specify the version field without specifying the image.
+
+If an upgrade fails, the Operator stops and reports the status of the failing
+component. Rolling your cluster back to a previous version is not supported.
```

Comment:
@kalexand-rh @abhinavdahiya Do we plan to state how to restore the cluster when hit a failed/partial upgrade since "Rolling your cluster back to a previous version is not supported" in this doc?

Reply:
If @abhinavdahiya can provide some information, I'm happy to write another topic to describe how to recover from a failed upgrade.